newproject/html.html

```diff
@@ -12,6 +12,7 @@
 
 <body>
 	<!--//header-->
+
 	<header>
```

Comment:
header shouldn't be hidden for small displays. You can create hamburger or change direction to column

Reply:
not fixed

Reply:
> not fixed

sry, didn't update branch. it was fixed